functions.php

```diff
@@ -54,12 +68,36 @@ function checkAllCards(): array {
     $queryString = 'SELECT * FROM `cards`';
     $query = $db->prepare($queryString);
     $query->execute();
-    $result = $query->fetchAll();
-    return $result;
-
+    return $result = $query->fetchAll();
 }
 
-function createMTGCard(array $result) {
+function createMTGCard(array $result): string {
```

Comment:
$result should have a more semantic name - result of what? and what does it contain? 
$card_array?